pubsub/gossipsub/README.md

```diff
@@ -424,7 +423,7 @@ GETNODES {}
 
 NODES {
  peers []peer.ID
- ttl int
+ TTL int
```

Comment:
These should probably be lower case for consistency.

Reply:
Ah sorry, that was from a Find and replace, I forgot about those.